testsuite/features/secondary/buildhost_docker_build_image.feature

```diff
@@ -87,24 +92,33 @@ Feature: Build container images
     And I wait at most 60 seconds until all "3" container images are built correctly on the Image List page
     And I wait at most 600 seconds until image "suse_real_key" with version "latest" is built successfully via API
     And I wait at most 300 seconds until image "suse_real_key" with version "latest" is inspected successfully via API
-    Then the list of packages of image "suse_real_key" with version "latest" is not empty
     When I wait until no Salt job is running on "build_host"
 
+@skip_if_github_validation
+  Scenario: Check the list of packages is not empty
+    Then the list of packages of image "suse_real_key" with version "latest" is not empty
+
```

Comment:
Could you please describe on the Pull Request description the reason why we must skip this step in the GH validation?

It gonna be important, so we can track it in the future.